operators/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(async_closure)]
+#![feature(available_concurrency)]
```

Comment:
Meh, it is required to activate some nightly feature for it? Are there alternatives?

Reply:
https://crates.io/crates/num_cpus

Reply:
Yeah, I've used this one in another project. I guess using it is one step closer to stable.
You should add a todo, though, that covers your feature that will come :tm: .

Reply:
87f62856d6517d1b2533690aeed904d4d9b1a43a